HW_6/app.py

```diff
@@ -0,0 +1,218 @@
+from pydantic import BaseModel
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.exc import IntegrityError
+from typing import List
+
+
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///shop.db'
+db = SQLAlchemy(app)
+
+
+class ProductCreate(BaseModel):
+    name: str
+    description: str
+    price: float
+
+class ProductRead(ProductCreate):
+    id: int
+
+class OrderCreate(BaseModel):
+    user_id: int
+    product_id: int
+    order_date: str
+    status: str
+
+class OrderRead(OrderCreate):
+    id: int
+
+class UserCreate(BaseModel):
+    first_name: str
+    last_name: str
+    email: str
+    password: str
+
+class UserRead(UserCreate):
+    id: int
+
+
+# Модель для таблицы товаров
+class Product(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255), nullable=False)
+    description = db.Column(db.Text, nullable=False)
+    price = db.Column(db.Float, nullable=False)
+
+# Модель для таблицы заказов
+class Order(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False)
+    product_id = db.Column(db.Integer, db.ForeignKey('product.id'), nullable=False)
+    order_date = db.Column(db.String(20), nullable=False)
+    status = db.Column(db.String(20), nullable=False)
+
+# Модель для таблицы пользователей
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    first_name = db.Column(db.String(100), nullable=False)
+    last_name = db.Column(db.String(100), nullable=False)
+    email = db.Column(db.String(255), unique=True, nullable=False)
+    password = db.Column(db.String(255), nullable=False)
+
+# Создание таблиц в базе данных
+db.create_all()
```

Comment:
Была ошибка тут. Нужно использовать 
```suggestion
with app.app_context():
    db.create_all()
```
У вас такой ошибки ен возникало?